src/app/components/substeps-details/substeps-details.component.html

```diff
@@ -1,2 +1,32 @@
-<!-- TODO fill out component template -->
-{{substep|json}}
+<mat-card-content>
+    <dl class="substep-details">
+        <div>
+            <dt>Cleared</dt>
+            <dd>
+                {{substep?.cleared}}
+            </dd>
+        </div>
+
+        <div>
```

Comment:
The ID is not something that needs (or should) to be displayed to the user (users can identify them by their sequence number)